dsc/src/subcommand.rs

```diff
@@ -411,20 +394,20 @@ pub fn resource(subcommand: &ResourceSubCommand, format: &Option<OutputFormat>,
                 table.print();
             }
         },
-        ResourceSubCommand::Get { resource, input, all } => {
+        ResourceSubCommand::Get { resource, input, path, all } => {
```

Comment:
Should we have a function that takes `input` (maybe needs to be renamed to `document` now?), `stdin`, and `path`, and just returns the string contents so that the `get`, `set`, etc... functions just take one `input` parameter instead of all of them?  You already have the `get_input()` below that can be used.